projects/fizzbuzz/java/README.md

```diff
@@ -1 +1,7 @@
 Place java solutions to fizz buzz in this directory
+
+To run this .java file:
+ open your java ide of choice. (If you dont have one, I recommend eclipse)
+ Next find the file tab and click open.
+ Navigate to where you have this java file downloaded and open it.
+ 
```

Comment:
code worked great. To improve: add more detail to the readme such as where teh option to run the program is actually located or how to create a run-able project (not sure if that required in eclipse but in intellij you must have a project set up to run not simply a file). Here is the output showing a successful test:
![output](https://user-images.githubusercontent.com/35410829/52234720-8d69f580-287f-11e9-8a07-332ccdec45af.JPG)